README.md

```diff
@@ -74,31 +74,40 @@ require("cmp_git").setup({
             filter = "all", -- assigned, created, mentioned, subscribed, all, repos
             limit = 100,
             state = "open", -- open, closed, all
+            sort_by = nil, -- nil, "number", "title", "body", or custom function
         },
         mentions = {
             limit = 100,
+            sort_by = nil, -- nil, "login", or custom function
         },
         pull_requests = {
             limit = 100,
             state = "open" -- open, closed, merged, all
+            sort_by = nil, -- nil, "number", "title", "body", or custom function
         }
     },
     gitlab = {
         issues = {
             limit = 100,
             state = "opened", -- opened, closed, all
+            sort_by = nil, -- nil, "iid", "title", "description", or custom function
         },
         mentions = {
             limit = 100,
+            sort_by = nil, -- nil, "username", "name", or custom function
         },
         merge_requests = {
             limit = 100,
             state = "opened", -- opened, closed, locked, merged
+            sort_by = nil, -- nil, "iid", "title", "description", or custom function
         },
     },
 })
 ```
 
+The custom function for sorting, takes the item (issue, pull/merge request or mention) as argument, and should
+return a string, which will be used to fill the `sortText` field.
+
```

Comment:
I think default sort should be `updated`. What do you think about it?

(I was planning to do it but you are fast than by me. Thank you!)

Reply:
I'll have a look. I'll keep you posted

Reply:
So I've worked a little on it now. It's note the most elegant ting in the world.

Notable is that to make it work, `cmp.sort.comparator.sort_text`´has to be called before `score`. If the overall design is nice, I'll add it to the readme